bitcoin/src/bip32.rs

```diff
@@ -6,8 +6,6 @@
 //! at <https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki>.
 //!
 
-#![allow(deprecated)] // Remove once we remove XpubIdentifier.
```

Comment:
FYI for the next time, globally allowing deprecated like this is not great since it will silence unrelated things.

Reply:
+1, though in this case it's a small module that's rarely touched.

Reply:
Yep, I couldn't get it to work directly on the type. And since it was one release I didn't put too much more thought into it. (It was an alias so slightly unusual.)